hospitalco/market/models.py

```diff
@@ -16,8 +26,11 @@ class Meta:
 class Requirement(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     user = models.OneToOneField(User, on_delete=models.CASCADE)
```

Comment:
also the Requirement should be linked to the `hospital_profile` rather than the `user`, right?

Reply:
This an upstream thing: pls report in #14.
I'm doing only as much work as is required to link items and categories.